MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -153,9 +153,8 @@ private IPlaybackManager CurrentPlaybackManager
         /// <returns></returns>
         public async Task Play(IEnumerable<IMediaFile> mediaFiles)
         {
-            var enumerable = mediaFiles as IList<IMediaFile> ?? mediaFiles.ToList();
             MediaQueue.Clear();
-            MediaQueue.AddRange(enumerable);
+            MediaQueue.AddRange(mediaFiles.ToList());
```

Comment:
The reason I left it there is because I thought it was there for a reason, namely to avoid multiple enumerations of the IEnumerable which might happen in the [`AddRange` method of `MediaQueue`](https://github.com/martijn00/XamarinMediaManager/blob/master/MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaQueue.cs#L169). When thinking about it again, it would make more sense to insert the `ToList()` into the `MediaQueue.AddRange()` method.